src/sentry/models/apitoken.py

```diff
@@ -42,11 +42,11 @@ class ApiToken(ReplicatedControlModel, HasApiScopes):
     user = FlexibleForeignKey("sentry.User")
     name = models.CharField(max_length=255, null=True)
     token = models.CharField(max_length=71, unique=True, default=generate_token)
-    hashed_token = models.CharField(max_length=128, null=True)
+    hashed_token = models.CharField(max_length=128, unique=True, null=True)
     token_type = models.CharField(max_length=7, choices=AuthTokenType, null=True)
     token_last_characters = models.CharField(max_length=4, null=True)
     refresh_token = models.CharField(max_length=71, unique=True, null=True, default=generate_token)
-    hashed_refresh_token = models.CharField(max_length=128, null=True)
+    hashed_refresh_token = models.CharField(max_length=128, unique=True, null=True)
```

Comment:
Just checking - are these unique, or do you just need indexes?

Reply:
They should be unique. Since authentication will use the hashed value eventually, if we had a collision here it would be ambiguous as to which user the particular API token belonged to.